app/bundles/MessengerBundle/MessageHandler/PageHitNotificationHandler.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\MessengerBundle\MessageHandler;
+
+use Mautic\LeadBundle\Entity\LeadRepository;
+use Mautic\MessengerBundle\Exceptions\InvalidPayloadException;
+use Mautic\MessengerBundle\Message\PageHitNotification;
+use Mautic\PageBundle\Entity\Hit;
+use Mautic\PageBundle\Entity\HitRepository;
+use Mautic\PageBundle\Entity\PageRepository;
+use Mautic\PageBundle\Entity\RedirectRepository;
+use Mautic\PageBundle\Model\PageModel;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Messenger\Handler\Acknowledger;
+use Symfony\Component\Messenger\Handler\MessageSubscriberInterface;
+
+class PageHitNotificationHandler implements MessageSubscriberInterface
```

Comment:
```suggestion
class PageHitNotificationHandler implements \Symfony\Component\Messenger\Handler\MessageHandlerInterface
```
There is no point in implementing `MessageSubscriberInterface` when we implement just the `__invoke()` method.